tests/manage/monitoring/conftest.py

```diff
@@ -431,3 +444,422 @@ def corrupt_pg():
     oc.delete(resource_name=dummy_deployment)
 
     return measured_op
+
+#
+# IO Workloads
+#
+
+
+@pytest.fixture
+def fio_pvc_dict():
+    """
+    PVC template for fio workloads.
+    Note that all 'None' values needs to be defined before usage.
+    """
+    template = textwrap.dedent("""
+        kind: PersistentVolumeClaim
+        apiVersion: v1
+        metadata:
+          name: fio-target
+        spec:
+          storageClassName: None
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: None
+        """)
+    pvc_dict = yaml.safe_load(template)
+    return pvc_dict
+
+
+@pytest.fixture
+def fio_configmap_dict():
+    """
+    ConfigMap template for fio workloads.
+    Note that you need to add actuall configuration to workload.fio file.
+    """
+    template = textwrap.dedent("""
+        kind: ConfigMap
+        apiVersion: v1
+        metadata:
+          name: fio-config
+        data:
+          workload.fio: |
+            # here comes workload configuration
+        """)
+    cm_dict = yaml.safe_load(template)
+    return cm_dict
+
+
+@pytest.fixture
+def fio_job_dict():
+    """
+    Job template for fio workloads.
+    """
+    template = textwrap.dedent("""
```

Comment:
why not have this yaml file in workloads dir?

Reply:
I tried not to change ocs_ci code unless necessary.

Moreover having these snippets in dedicated files wouldn't be much useful, as it's purpose is to be loaded into python dictionary, tweaked a bit and deployed.

Reply:
I agree with Vasu. IMO, it having this in a yaml file will make the code here clearer and allow other places in the project to use it

Reply:
+1 on yaml.

Also apply on 2 fixtures above

Reply:
The expected way to use these snippets is via fixtures, and can be used by all tests in `tests/manage/monitoring` directory tree. To allow other places to use it, I could move it config.py one or two levels up.

Reply:
@mbukatov I still think it would be better to keep consistency with how we usually deal with yamls and save this in a dedicated file under workloads. 
With that said, I can live with having it inside the fixture :)

Reply:
ok